stdlib/public/core/KeyPath.swift

```diff
@@ -3703,3 +3703,138 @@ internal func _instantiateKeyPathBuffer(
   }
 }
 
+@_silgen_name("swift_keypath_dladdr")
+internal func keypath_dladdr(_: UnsafeRawPointer) -> UnsafePointer<CChar>?
+
+@_silgen_name("swift_keypathSourceString")
+internal func demangle(
+    name: UnsafePointer<CChar>
+) -> UnsafePointer<CChar>?
+
+
+fileprivate func dynamicLibraryAddress<Base, Leaf>(
+    of pointer: ComputedAccessorsPtr,
+    _: Base.Type,
+    _ leaf: Leaf.Type
+) -> String {
+    let getter: ComputedAccessorsPtr.Getter<Base, Leaf> = pointer.getter()
+    let pointer = unsafeBitCast(getter, to: UnsafeRawPointer.self)
+    if let cString = keypath_dladdr(UnsafeRawPointer(pointer)),
+        let demangled = demangle(name: cString) {
+        defer {
+            demangled.deallocate()
+        }
+        return String(cString: demangled)
+    } else {
+        return "<computed \(pointer) (\(leaf))>"
+    }
+}
```

Comment:
I think all of this and the actual implementation in `debugDescription` should be inside a `#if SWIFT_ENABLE_REFLECTION`.

Reply:
I just want to confirm, if `SWIFT_ENABLE_REFLECTION` is off, we should print `"(value cannot be printed without reflection)"`, right?